explainaboard/tests/test_machine_translation.py

```diff
@@ -75,6 +76,31 @@ def test_default_features_dont_modify_condgen(self):
         # condgen features are a subset of MT features
         self.assertDictEqual(mt_features, {**mt_features, **condgen_features_1})
 
+    def test_custom_features(self):
+        loader = get_custom_dataset_loader(
```

Comment:
Actually, this type of test for `customized features` is relatively shallow (That's part of the reason why we didn't capture the bug of not using `customized` features). A better idea is to see if this feature has been put into use in the final analysis report. But this could be done in future PR.